test/MarketCollateralPool.js

```diff
@@ -21,95 +21,237 @@ contract('MarketCollateralPool', function(accounts) {
   const entryOrderPrice = 33025;
```

Comment:
some of these vars are no longer used, can we clean them up?

Reply:
Sure, cleaning those up now.